sources/packages/backend/workflow/test/2022-2023/assessment-gateway/assessment-gateway.originalAssessment.e2e-spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ASSESSMENT_ID } from "@sims/services/workflow/variables/assessment-gateway";
+import {
+  ApplicationExceptionStatus,
+  AssessmentTriggerType,
+} from "@sims/sims-db";
+import { ZBClient } from "zeebe-node";
+import { AssessmentConsolidatedData } from "../../models";
+import {
+  createFakeConsolidatedFulltimeData,
+  ZeebeMockedClient,
+  PROCESS_INSTANCE_CREATE_TIMEOUT,
+  WorkflowServiceTasks,
+  createFakeSingleIndependentStudentData,
+  expectToPassthroughServiceTasks,
+} from "../../test-utils";
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+
+describe(`E2E Test Workflow assessment gateway on original assessment for ${PROGRAM_YEAR}`, () => {
+  let zeebeClientProvider: ZBClient;
+  beforeAll(async () => {
+    zeebeClientProvider = ZeebeMockedClient.getMockedZeebeInstance();
+  });
+
+  it("Should follow the expected workflow path when student is single and independent without application exception and PIR.", async () => {
+    // Arrange
+    const assessmentConsolidatedData: AssessmentConsolidatedData = {
+      ...createFakeConsolidatedFulltimeData(PROGRAM_YEAR),
+      assessmentTriggerType: AssessmentTriggerType.OriginalAssessment,
+      ...createFakeSingleIndependentStudentData(),
+      // Application with no exception.
+      applicationExceptionStatus: ApplicationExceptionStatus.Approved,
```

Comment:
Is this data not expected to be mocked on `verify-application-exceptions-task-result`?